.circleci/config.yml

```diff
@@ -77,12 +77,19 @@ jobs:
             git checkout master
             CURR_MBT_VERSION=$(cat ./VERSION)
             echo "Current release: ${CURR_MBT_VERSION}"
-            if [[-z "$MBT_VERSION_MANUAL"]]; then 
-              echo "Using manual MBT version ${MBT_VERSION_MANUAL}"
-              MBT_VERSION=$MBT_VERSION_MANUAL
-            else
-              MBT_VERSION=$(echo $CURR_MBT_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
-            fi
+            MBT_VERSION=$(echo $CURR_MBT_VERSION |awk -F. -v MAJOR_VAR="$MAJOR" -v MINOR_VAR="$MINOR" -v PATCH_VAR="$PATCH" '{
+                                                    if (MAJOR_VAR == "true"){
+                                                      $1++;
```

Comment:
The semicolons aren't necessary. This is from the `awk` documentation about statements:

> Normally, a statement ends with a newline, however, this is not the case for lines ending in a ",", **{**, **?**, **:**, **&&**, or **||**. Lines ending in **do** or **else** also have their statements automatically continued on the following line. In other cases, a line can be continued by ending it with a "\\", in which case the newline will be ignored.
> Multiple statements may be put on one line by separating them with a ";".

(From https://linux.die.net/man/1/awk)